pennylane_cirq/cirq_device.py

```diff
@@ -108,12 +108,12 @@ def __init__(self, wires, shots, qubits=None):
 
         # Add inverse operations
         self._inverse_operation_map = {}
-        for key in self._operation_map:
-            if not self._operation_map[key]:
+        for key, value in self._operation_map.items():
+            if not value:
```

Comment:
Was the reason to make these changes improving code quality?

Reply:
No CodeFactor was not happy about it and was failing...